core/chains/evm/config/toml/defaults/fallback.toml

```diff
@@ -26,6 +26,22 @@ ResendAfterThreshold = '1m'
 [BalanceMonitor]
 Enabled = true
 
+[ClientErrors]
+NonceTooLow = 'nonce too low'
+NonceTooHigh = 'nonce too high'
+ReplacementTransactionUnderpriced = 'replacement transaction underpriced'
+LimitReached = 'limit reached'
+TransactionAlreadyInMempool = 'transaction already in mempool'
+TerminallyUnderpriced = 'terminally underpriced'
+InsufficientEth = 'infufficient Eth'
+TxFeeExceedsCap = 'tx fee exceeds cap'
+L2FeeTooLow = 'l2 fee too low'
+L2FeeTooHigh = 'l2 fee too high'
+L2Full = 'l2 full'
+TransactionAlreadyMined = 'transaction already mined'
+Fatal = 'fatal'
+ServiceUnavailable = 'service unavailable'
```

Comment:
Shouldn't these be regular expressions, not plain strings?

Reply:
Yeah I'll add a matching prefix

Reply:
If we are going to have these defined in the TOML, then they need to be _exactly_ the same as the strings defined in code already.

Reply:
Agreed, This is outdated, they are the same as in the code now.

Reply:
Please double check them. The prefixes were not the only differences.

Reply:
I will double check them